code/components/extra-natives-five/src/VehicleExtraNatives.cpp

```diff
@@ -371,9 +381,37 @@ static HookFunction initFunction([]()
 		auto location = hook::get_pattern<char>("F6 83 ? ? ? ? 07 75 ? 44 0F");
 
 		CurrentRPMOffset = *(uint32_t*)(location - 42);
+		ClutchOffset = CurrentRPMOffset + 12;
 		ThrottleOffsetOffset = CurrentRPMOffset + 16;
 	}
 
+	{
+		auto location = hook::get_pattern<char>("45 0f 57 ? f3 0f 11 ? ? ? 00 00 f3 0f 5c");
```

Comment:
Can you please uppercase all pattern bytes?

Reply:
My mistake, didn't even notice. Will fix.